src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,111 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+
+1.15.0.dev3 (3/9/2019)
+----------------------
+
+New Features
+~~~~~~~~~~~~
+* Publish the awslambda contrib to PyPI. (#7343)
+  `PR #7343 <https://github.com/pantsbuild/pants/pull/7343>`_
+
+* Add subsystem for appending tags to targets from a single source (#7315)
+  `PR #7315 <https://github.com/pantsbuild/pants/pull/7315>`_
+
+Bugfixes
+~~~~~~~~
+* Revert "add a lint to warn on implicit string concatenation (#7286)" (#7340)
+  `PR #7286 <https://github.com/pantsbuild/pants/pull/7286>`_
+
+* Fix ./pants2 overriding desired $PY interpreter (#7338)
+  `PR #7338 <https://github.com/pantsbuild/pants/pull/7338>`_
+
+* Fix regression to python interpreter selection test from #7257 (#7333)
+  `PR #7333 <https://github.com/pantsbuild/pants/pull/7333>`_
+
+* Fix ./pants2 overriding preset interpreter constraints (#7334)
+  `PR #7334 <https://github.com/pantsbuild/pants/pull/7334>`_
+
+* Bugfix for stale interpreter.info purging when binary exe is no longer on the filesystem (#7292)
+  `PR #7292 <https://github.com/pantsbuild/pants/pull/7292>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+* Specify ABI for pantsbuild.pants wheel and build with both UCS2 and UCS4 (#7235)
+  `PR #7235 <https://github.com/pantsbuild/pants/pull/7235>`_
+
+* Propagate global interpreter constraints when building PEXes with interpreter constraints requested (#7285)
```

Comment:
We might want to classify this as an API change. It impacts how `./pants binary` works and could surprise users. Earlier it would only add interpreter constraints to the PEX if the target had any; now it will default to global constraints if the target does not have any.